src/modules/registration/styles/Registration.module.css

```diff
@@ -26,12 +26,16 @@
     border-radius: 3.125em;
     font-size: x-large;
     color: #fff;
+    cursor: pointer;
 }
 
 .returnToLoginButton {
     margin-bottom: 4em;
+    width: fit-content;
```

Comment:
Interesting, i never use this and there is probably a better way to do what you are trying to do, but ima merge this anyway and DM you later when i know what it is

Reply:
The button was the width of the screen but had no background and no border, so I wanted the button to be the width of the text only.